_security/configuration/demo-configuration.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: default
+title: Demo Configuration Setup Guide
+parent: Configuration
+nav_order: 4
+redirect_from:
+ - /security-plugin/configuration/demo-configuration/
```

Comment:
I would need doc-team's help in setting up proper navigation to this setup guide.

Reply:
@DarshitChanpura: We likely don't need the redirect here since this is a new page, unless you are trying to redirect from a previously existing page.

Reply:
oh okay. I understood it as a path of the current page and that's why got confused on how it would be used. I can remove this.